nectar/downloaders/threaded.py

```diff
@@ -160,6 +160,7 @@ def requests_kwargs_from_nectar_config(config):
                                          host=parsed_url.host, port=config.proxy_port)
             requests_kwargs['proxies'] = {'http': parsed_url.url, 'https': parsed_url.url}
 
+        requests_kwargs['stream'] = True # required for reading the download in chunks
```

Comment:
I would suggest doing this in the streamer:
```
diff --git a/streamer/pulp/streamer/server.py b/streamer/pulp/streamer/server.py
index 7dc3acb..ac8e72c 100644
--- a/streamer/pulp/streamer/server.py
+++ b/streamer/pulp/streamer/server.py
@@ -117,6 +117,7 @@ class Streamer(Resource):
         # to avoid carrying the package.
         self.session = Session()
         self.session.mount('https://', pulp_adapters.PulpHTTPAdapter())
+        self.session.stream = True
 
     def render_GET(self, request):
         """
```